### ISSUE TITLE
sitemap.default.xml is reference missing in sitemap.xml when using SitemapController route

### ISSUE THREAD

**Issue by ostrolucky** [prestaconcept/PrestaSitemapBundle#198]
Repro steps:
- create new symfony project
- install bundle
- add 

```
presta_sitemap:
    resource: "@PrestaSitemapBundle/Resources/config/routing.yml"
```

to routing file

- Add `@Route("/foo", name="foo", options={"sitemap"=true})` to some action
- go to sitemap.xml. result: blank xml
- go to sitemap.default.xml. result: there is the route

Expected: there should be reference to sitemap.default.xml from within sitemap.xml

**Comment by crabnky** [comment]
+1

**Comment by yann-eugone** [comment]
Not a solution but downgrading to `1.5.4` fixed the issue.

I'm having a look to the commit that introduced this issue.

**Comment by yann-eugone** [comment]
Found that https://github.com/prestaconcept/PrestaSitemapBundle/commit/06b06613f7faa65aec26f2160d780500ca077274#diff-39b1b46dd9275aacb363c1d5a66ae48dR93 has introduced this.

**Comment by yann-eugone** [comment]
@ostrolucky @crabnky can you please test that #200 fixed the issue on your projects ?

`composer require presta/sitemap-bundle:dev-issue-198.bugfixed-sitemap-index-generation@dev`

@ostrolucky please check that this has no effect on what you've done in #181

If everything is fine i will merge + tag that day

**Comment by ostrolucky** [comment]
Fixes the issue 👍

**Comment by yann-eugone** [comment]
Released in [1.7.0](https://github.com/prestaconcept/PrestaSitemapBundle/releases/tag/v1.7.0)

**Comment by crabnky** [comment]
Thanks, all fine now.
